### PR TITLE
Revise formatting of READMEs.

### PR DESCRIPTION
**Pull request by awholegunch**

### CODE REVIEW

File: cdap-stream-clients/java/README.md [hunk 1]
```diff
@@ -1,91 +1,95 @@
...
-```
- 
-#### Create Stream
+
+- ssl: false (set true to use HTTPS protocol)
+- verifySSLCert: true (set false to suspend certificate checks; this allows self-signed certificates when SSL is true)
+- authClient: null ([CDAP Authentication Client](https://github.com/caskdata/cdap-clients/tree/develop/cdap-authentication-clients/java)
+  to interact with a secure CDAP instance)
+
+    StreamClient streamClient = new RestStreamClient.Builder("localhost", 10000)
```
Comment (anwar6953) [inline cdap-stream-clients/java/README.md:46]: This is no longer in a code block, so it more difficult to parse.

File: cdap-stream-clients/python/README.md [hunk 2]
```diff
@@ -48,87 +48,78 @@ Optional configurations that can be set (and their default values):
...
-  config.ssl = True
-  config.set_auth_client(authentication_client)
-
-  stream_client = StreamClient(config)
-```
-
-#### Create Stream
+  to interact with a secure CDAP instance)
+
+    config = Config()
```
Comment (anwar6953) [inline cdap-stream-clients/python/README.md:56]: This is no longer in a code block, and so it is hard to parse.
image

**Comment by awholegunch** [comment]
Fixed formatting. Please review again.

**Comment by anwar6953** [comment]
:+1:
